transcom/mymove
# Int b 21671 db create re fsc multipliers

## [B-21671](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1031069)

## Summary

We are adding FSC multipliers table that will be used in the calculations for the price of the FSC.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Test that the migrations work using `make db_dev_migrate`
2. Verify the new table with the populated columns: id, low_weight, high_weight, multiplier, created_at, and updated_at. (I used Dbeaver)

### Database

#### Any new migrations/schema changes:
